scipy/stats/_multivariate.py

```diff
@@ -1230,7 +1231,6 @@ def _dirichlet_check_parameters(alpha):
 
 
 def _dirichlet_check_input(alpha, x):
-    x = np.asarray(x)
```

Comment:
`dirichlet` and `multivariate_beta` both have a new `_check_input` method. 
`dirichlet` does `x = np.asarray(x)` then calls `_dirichlet_check_input(alpha, x)`.
`multivariate_beta` does `x = np.moveaxis(x, -1, 0)` then calls  `_dirichlet_check_input(alpha, x)`.
That is the only thing that's different between the two distributions, other than documentation.